bin/masspay.py

```diff
@@ -193,6 +193,11 @@ def post_back_to_gratipay():
     except KeyError:
         gratipay_base_url = 'https://gratipay.com'
 
+    nmasspays = int(requests.get(gratipay_base_url + '/dashboard/nmasspays').text())
+    if nmasspays < 10:
```

Comment:
Sounds good. Would it be helpful to add some instructions in the print out below, like "if this is a false alarm, please modify xxx manually and rerun"?